src/SPCPAuthClient.class.ts

```diff
@@ -316,18 +343,18 @@ class SPCPAuthClient {
 // Functions for extracting attributes from Artifact Response
 SPCPAuthClient.extract = {
   CORPPASS: ([element]) => {
-    const cpXMLBase64 = xpath.select('string(./*[local-name(.)=\'AttributeValue\'])', element)
+    const cpXMLBase64 = xpath.select('string(./*[local-name(.)=\'AttributeValue\'])', element) as unknown as string
```

Comment:
so what I think is happening here is that `cpXMLBase64` is really a `string[]`, but `base64.decode` [converts it to a string](https://github.com/mathiasbynens/base64/blob/913b89753d99362855c71c02d04384d1d1a9c2fd/src/base64.js#L41). this casting is as close as I could get without declaring our own types for `base-64`.

Reply:
I guess we really want a string, so how about...
```suggestion
    const cpXMLBase64 = xpath.select('string(./*[local-name(.)=\'AttributeValue\'])', element, true)!.toString()
```